clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/provider/agent/AbstractAppengineCachingAgent.groovy

```diff
@@ -40,6 +40,30 @@ abstract class AbstractAppengineCachingAgent implements CachingAgent, AccountAwa
     this.credentials = credentials
   }
 
+  boolean shouldIgnoreLoadBalancer(String loadBalancerName) {
+    if (credentials.services != null && !credentials.services.isEmpty() &&
+      credentials.services.find { loadBalancerName.matches(it) } == null) {
```

Comment:
Could be an `.any` (or an `.every`)

Reply:
✅ Updated `== null` to be `.every` and `!= null` to be `.any`.